src/main/java/core/basesyntax/Areable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Areable {
+    public abstract double getArea();

```

Comment:
```suggestion
    double getArea();
```
it is public and abstract by default